roc/address.go

```diff
@@ -0,0 +1,73 @@
+package roc
+
+/*
+#cgo LDFLAGS: -lroc
+#include <roc/receiver.h>
+#include <roc/address.h>
+#include <roc/sender.h>
+#include <roc/log.h>
+#include <stdlib.h>
```

Comment:
Could you please cleanup C includes here and in other places? I don't think every go file needs all of them. Also, to we need `-lroc` in every file?

Reply:
I need to investingate where the `-lroc` is needed, but otherwise includes are removed